frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -0,0 +1,49 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '../protractor.conf';
+import * as loginView from '../views/login.view';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 3;
+
+describe('Auth test', () => {
+  beforeAll(async() => {
+    await browser.get(appHost);
+  });
+
+  const {BRIDGE_AUTH_USERNAME, BRIDGE_AUTH_PASSWORD} = process.env;
+  if (BRIDGE_AUTH_USERNAME && BRIDGE_AUTH_PASSWORD) {
+    describe('Login test', async() => {
+      beforeAll(() => {
+        // Extend the default jasmine timeout interval just in case it takes a while for the htpasswd idp to be ready
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+      });
+
+      afterAll(() => {
+        // Set jasmine timeout interval back to the original value after these tests are done
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_DEFAULT_TIMEOUT_INTERVAL;
+      });
+
+      it('logs in as "test" user using htpasswd identity provider', async() => {
+        await loginView.login('test', 'test', 'test');
+        expect(loginView.userDropdown.isDisplayed()).toBeTruthy();
+        expect(loginView.userDropdown.getText()).toContain('test');
+      });
+
+      it('logs out "test" user', async() => {
+        await loginView.logout();
+        expect($('.login-pf').isPresent()).toBeTruthy();
+      });
+
+      it('logs in as kubeadmin user', async() => {
+        await loginView.login('kube:admin', BRIDGE_AUTH_USERNAME, BRIDGE_AUTH_PASSWORD);
+        expect(loginView.userDropdown.isDisplayed()).toBeTruthy();
+        expect(loginView.userDropdown.getText()).toContain('kube:admin');
+      });
+    });
+  }
+  it('displays kubeadmin user global notification', async() => {
```

Comment:
I'd probably put this inside the `if`. We want to be able to run tests as users other than kubeadmin.